tutorials/README.md

```diff
@@ -0,0 +1,168 @@
+
+Once you have familiarised yourself with the methods and identified how to apply SBI to your use case, ensure you work through the **Diagnostics** tutorials linked below, to identify failure cases and assess the quality of your inference.
+
+
+## I develop methods for `sbi` (_contributer_)
+
+### Introduction
+- [Getting started](00_getting_started_flexible.ipynb) introduces the `sbi` package and its core functionality.
+- [Inferring parameters for multiple observations ](01_gaussian_amortized.ipynb)introduces the concept of amortization.
+- [All implemented methods](16_implemented_methods.ipynb) provides an overview of the implemented inference methods and how to call them.
+
+### Advanced:
+- [Multi-round inference](03_multiround_inference.ipynb)
+- [Sampling algorithms in sbi](11_sampler_interface.ipynb)
+- [Custom density estimators](04_density_estimators.ipynb)
+- [Learning summary statistics](05_embedding_net.ipynb)
+- [SBI with trial-based data](14_iid_data_and_permutation_invariant_embeddings.ipynb)
+- [Handling invalid simulations](08_restriction_estimator.ipynb)
+- [Crafting summary statistics](10_crafting_summary_statistics.ipynb)
+
+### Diagnostics:
+- [Posterior predictive checks](12_diagnostics_posterior_predictive_check.ipynb)
+- [Simulation-based calibration](13_diagnostics_simulation_based_calibration.ipynb)
+- [Density plots and MCMC diagnostics with ArviZ](15_mcmc_diagnostics_with_arviz.ipynb)
+
+### Analysis:
+- [Conditional distributions](07_conditional_distributions.ipynb)
+- [Posterior sensitivity analysis](09_sensitivity_analysis.ipynb)
+### Examples:
+- [Hodgkin-Huxley example](../examples/00_HH_simulator.ipynb)
+- [Decision making model](../examples/01_decision_making_model.ipynb)
+
+Please first read our [contributer guide](../CONTRIBUTING.md) and our [code of conduct](../CODE_OF_CONDUCT.md).
+
+
+
+
+## Overview
+
+
+`sbi` lets you choose from a variety of _amortized_ and _sequential_ SBI methods:
+
+Amortized methods return a posterior that can be applied to many different observations without retraining,
+whereas sequential methods focus the inference on one particular observation to be more simulation-efficient.
+For an overview of implemented methods see below, or checkout our [GitHub page](https://github.com/mackelab/sbi).
+
+- To learn about the general motivation behind simulation-based inference, and the
+  inference methods included in `sbi`, read on below.
+
+- For example applications to canonical problems in neuroscience, browse the recent
+  research article [Training deep neural density estimators to identify mechanistic models of neural dynamics](https://doi.org/10.7554/eLife.56261).
+
+
+
+## Motivation and approach
+
+Many areas of science and engineering make extensive use of complex, stochastic,
+numerical simulations to describe the structure and dynamics of the processes being
+investigated.
+
+A key challenge in simulation-based science is constraining these simulation models'
```

Comment:
"simulation-based science" is a bit wierd formulation.

Maybe something like "simulation models for science is constraining the parameters of these models"...

Reply:
good then i will also  change this in the docs for the website - this text is from there